### PR TITLE
Bug 1828280: Remove deprecated flags and commands from test-cmd

### PR DESCRIPTION
**Pull request by soltysh**
To land https://github.com/openshift/oc/pull/310 we need to update the tests to drop the deprecated flags or change to use the currently available. 

/assign @sallyom

### CODE REVIEW

File: test/extended/testdata/cmd/test/cmd/policy-storage-admin.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ os::cmd::expect_failure_and_text 'oc policy --as=storage-adm remove-user screele
 #os::cmd::expect_success 'oc login -u storage-adm2 -p pw'
 #os::cmd::expect_success_and_text 'oc whoami' "storage-adm2"
 os::cmd::expect_success 'oc new-project --as=storage-adm2 --as-group=system:authenticated:oauth --as-group=system:authenticated policy-can-i'
-os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create pod --all-namespaces' 'no'
+os::cmd::expect_failure_and_text 'oc policy --as=storage-adm2 can-i create pod --all-namespaces' 'no'
 os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create pod' 'yes'
 os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create pvc' 'yes'
 os::cmd::expect_success_and_text 'oc policy --as=storage-adm2 can-i create endpoints' 'yes'
```
Comment (sallyom) [inline test/extended/testdata/cmd/test/cmd/policy-storage-admin.sh:45]: these should be 'oc auth can-i' also, right?

File: test/extended/testdata/cmd/test/cmd/volumes.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ os::test::junit::declare_suite_start "cmd/volumes"
 os::cmd::expect_success 'oc create -f ${TEST_DATA}/test-deployment-config.yaml'
 os::cmd::expect_success 'oc create -f ${TEST_DATA}/rollingupdate-daemonset.yaml'
 
-os::cmd::expect_success_and_text 'oc set volume dc/test-deployment-config --list' 'vol1'
+os::cmd::expect_success_and_text 'oc set volume dc/test-deployment-config' 'vol1'
 os::cmd::expect_success 'oc set volume dc/test-deployment-config --add --name=vol0 -m /opt5'
```
Comment (sallyom) [inline test/extended/testdata/cmd/test/cmd/volumes.sh:20]: i see a few more `--list` flags below, L31,33,36,38

File: test/extended/testdata/cmd/test/cmd/policy-storage-admin.sh (hunk 1 above)
Comment (soltysh) [reply test/extended/testdata/cmd/test/cmd/policy-storage-admin.sh:45]: Good catch!

File: test/extended/testdata/cmd/test/cmd/volumes.sh (hunk 2 above)
Comment (soltysh) [reply test/extended/testdata/cmd/test/cmd/volumes.sh:20]: Good catch!

**Pull request merged**